components/layout/layout.tsx

```diff
@@ -38,13 +38,9 @@ function generator({ suffixCls, tagName }: GeneratorProps) {
 
 class Basic extends React.Component<BasicProps, any> {
   render() {
-    const { prefixCls, className, children, tagName: CustomElement, ...others } = this.props;
+    const { prefixCls, className, children, tagName, ...others } = this.props;
     const classString = classNames(className, prefixCls);
-    return (
-      <CustomElement className={classString} {...others}>
-        {children}
-      </CustomElement>
-    );
+    return React.createElement(tagName, { className: classString, ...others }, children);
```

Comment:
Just make `tagName` def to `tagName?`. This update may not resolve #15162.

Reply:
Oh no, I forgot to change `tagName` to `tagName?`.